src/data/markdown/docs/03 cloud/05 Project and Team Management/02 Teams.md

```diff
@@ -0,0 +1,55 @@
+---
+
+title: 'Teams'
+excerpt: 'Guide on setting up Teams in k6 Cloud'
+---
+
+> ⭐️ Teams are available on Enterprise plans.
+
+## What are Teams?
+
+A _Team_ is an optional organizational level sitting between [Organizations](/cloud/project-and-team-management/organizations) and [Projects](/cloud/project-and-team-management/projects). With Teams, you can group the Projects of your Organizations into categories. All Team members gain access to the Projects under that Team. The Team can provide varying [Membership](/cloud/project-and-team-management/members) levels to each Project.
+
+Using Teams can help when administrating k6 Organizations with a large number of members and projects. For example, you could have a "Backend" Team that grants read-write access to all backend-related k6-cloud Projects and a "Payment" Team dealing only with Projects testing payment-related services. The Organization-admin can add/remove a member from the Team rather than having to do so for every related Project. 
```

Comment:
```suggestion
Teams help you administrate k6 Organizations with a large number of members and projects.
For example, a "Backend" Team could grant read-write access to all backend-related k6-cloud Projects,
and a "Payment" Team might deal only with Projects for testing payment-related services.
In this case, an Organization admin could add or remove members from these Teams (rather than doing so for each project). 
```